bin/lint-staged.js

```diff
@@ -103,6 +121,8 @@ const options = {
   relative: !!cliOptions.relative,
   shell: cliOptions.shell /* Either a boolean or a string pointing to the shell */,
   stash: !!cliOptions.stash, // commander inverts `no-<x>` flags to `!x`
+  hidePartiallyStaged:
+    cliOptions.hidePartiallyStaged == null ? !!cliOptions.stash : !!cliOptions.hidePartiallyStaged, // commander inverts `no-<x>` flags to `!x`
```

Comment:
Can the flag be `null` or is only ever `undefined`? It would be nice to use strict equality `===` if possible.

Reply:
Hmm, would this be equivalent to:

```suggestion
  hidePartiallyStaged:
    cliOptions.hidePartiallyStaged ?? !!cliOptions.stash, // commander inverts `no-<x>` flags to `!x`
```